data/pools/statusCache.go

```diff
@@ -62,6 +62,6 @@ func (sc *statusCache) put(tx transactions.SignedTxn, txErr string) {
 }
 
 func (sc *statusCache) reset() {
-	sc.cur = map[transactions.Txid]statusCacheEntry{}
-	sc.prev = map[transactions.Txid]statusCacheEntry{}
+	sc.cur = make(map[transactions.Txid]statusCacheEntry, sc.sz)
+	sc.prev = make(map[transactions.Txid]statusCacheEntry, sc.sz)
```

Comment:
Hm, maybe prev should become size 0, or even nil? Since it's never assigned to directly, I don't see why it needs to be initialized to `sc.sz`

Reply:
True! Will change.